{"name": "test_vwo_login_negative", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestVWOLogin' object has no attribute 'username'", "trace": "self = <tests.vwoLoginTests.test_vwo_login_pf.TestVWOLogin object at 0x000001D639FB58B0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"70487c0c055c9f276244d98016fcd1f7\")>\n\n    @pytest.mark.usefixtures(\"setup\")\n    @pytest.mark.qa\n    def test_vwo_login_negative(self, setup):\n        driver = setup\n        driver.get(Constants.app_url())\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(usr=self.username, pwd=\"123\")\nE       AttributeError: 'TestVWOLogin' object has no attribute 'username'\n\ntests\\vwoLoginTests\\test_vwo_login_pf.py:20: AttributeError"}, "start": 1727078541032, "stop": 1727078549192, "uuid": "fd7f4fbb-25f1-4177-8a6b-c842d8373522", "historyId": "412ef7bf0164aaf5de5dd43bf3faee91", "testCaseId": "412ef7bf0164aaf5de5dd43bf3faee91", "fullName": "tests.vwoLoginTests.test_vwo_login_pf.TestVWOLogin#test_vwo_login_negative", "labels": [{"name": "feature", "value": "Login Test"}, {"name": "epic", "value": "VWO App"}, {"name": "tag", "value": "qa"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login_pf"}, {"name": "subSuite", "value": "TestVWOLogin"}, {"name": "host", "value": "LAPTOP-BELRSIMA"}, {"name": "thread", "value": "11176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login_pf"}]}